pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -89,9 +90,9 @@ protected void setupTenants()
       throws IOException {
   }
 
-  @Override
-  protected boolean useMultiStageQueryEngine() {
-    return true;
+  @BeforeTest
+  protected void beforeTest() {
+    setUseMultiStageQueryEngine(true);
```

Comment:
This test was incorrect. It was always running as multi stage while some test actually call `setUseMultiStageQueryEngine` in order to do some things with V1